packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -22,9 +22,10 @@ URL:     https://theforeman.org/plugins/katello
 Source0: https://rubygems.org/downloads/%{gem_name}-%{version}%{?prerelease}.gem
 
 Requires: foreman-postgresql
+Requires: foreman < %{foreman_max_version}
+BuildRequires: %{?scl_prefix}rubygem(pg)
```

Comment:
@jlsherrill seems this was introduced by https://github.com/Katello/katello/commit/514770541975012ea85a5780fd67c237cb535e2d but never added a dependency to the gemspec

Reply:
ahh, did not realize this was a build time dep now, but it makes sense

Reply:
https://github.com/Katello/katello/pull/8757